src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -1051,6 +1064,13 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
                                 clearable={false}
                                 searchable={true}
                             />
+                            { dataSourceDescription && (
```

Comment:
just want to confirm that it in fact makes sense to render the menu even before dataSourceDescription has loaded and then re-render it when it loads.

Reply:
fixed